terminal.go

```diff
@@ -542,6 +548,11 @@ func (g *LightningTerminal) start() error {
 		return fmt.Errorf("could not connect to LND")
 	}
 
+	// In order to be able to create unique middleware request identifiers,
+	// we set a new unique connection ID. This should be refreshed every
+	// time we (re)connect to LND.
+	g.lndConnID = randId(LndConnIdLen)
```

Comment:
nit: maybe it makes more sense to add this in the `connectLND` function instead? Then you can perhaps add it as a return value of the function.

Reply:
right, I considered that, but I thought better to keep `connectLND` free from side effects and only assign to `g` in this scope